tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -39,14 +41,18 @@ bool IsRetriable(Status status) {
 
 Status CallWithRetries(const std::function<Status()>& f) {
   int attempts = 0;
+  int delay;
```

Comment:
I wanted to reuse the delay variable in each loop instead of creating a new one every time.

Should I still go ahead and make the change?